components/inputs/docs/input-checkbox.md

```diff
@@ -1,51 +1,78 @@
-# Checkboxes
+# Checkbox Input
 
-Checkboxes are used in forms to toggle an option or preference.
+Checkboxes are used in forms to toggle an option or preference. They may be grouped to form a set of options.
 
-<!-- docs: demo -->
+<!-- docs: demo display:block -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/inputs/input-checkbox.js';
+  import '@brightspace-ui/core/components/inputs/input-checkbox-group.js';
 </script>
-<d2l-input-checkbox label="Label for checkbox" checked></d2l-input-checkbox>
+<d2l-input-checkbox-group label="Toppings">
+  <d2l-input-checkbox label="Ketchup" checked></d2l-input-checkbox>
+  <d2l-input-checkbox label="Mustard"></d2l-input-checkbox>
+  <d2l-input-checkbox label="Relish"></d2l-input-checkbox>
+</d2l-input-checkbox-group>
 ```
 
 ## Best Practices
 <!-- docs: start best practices -->
 <!-- docs: start dos -->
 * Use in a form to indicate an option or preference.
-* Use to allow the user to select multiple, independent options from a set
-* Use an indeterminate state to indicate a mixed state where some child items are checked and some are not
+* Use to allow the user to select multiple, independent options from a set.
+* When multiple options are presented together, [wrap them in a group](#d2l-input-checkbox-group).
+* Use an indeterminate state to indicate a mixed state where some child items are checked and some are not.
 * Use as progressive disclosure in forms, so long as users are made aware both visually and non-visually that new options have been made available.
 <!-- docs: end dos -->
 
 <!-- docs: start donts -->
 * Don't use as a toggle that performs an immediate action, use a Switch component.
-* Don't use for mutually exclusive options, use radio buttons.
+* Don't use for mutually exclusive options, use radio input groups.
```

Comment:
Here too, with the period.

We could also link to the radio input page? And the switch page while we're at it.

Reply:
I've removed the periods. I tried linking to the radio input page but all it did was link to the page in core's repo, not within the Daylight Site. Maybe this isn't possible?